### ISSUE TITLE
Remove title attributes from the Classic Editor warning

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#8315]
Splitting this out from #8247 

The buttons in the Classic Editor warning use title attributes.

We shouldn't introduce new `title` attributes in core, especially after all the work done to remove most of them, see https://core.trac.wordpress.org/query?keywords=~title-attribute

Title attributes are not available to keyboard users, and may not be announced by screen readers depending on user settings. For more details please refer to https://core.trac.wordpress.org/ticket/24766

UI controls should be labelled in a way that's clear without requiring additional (semi-hidden) information that's not available to all users.

Aside: it would be nice to add a `type="button"` to the `<button>` element used in the modal.

**Comment by aduth** [comment]
Apologies for missing this in #8247. I assumed the markup there was inherited from other use of `.notification-dialog-wrap` in core. Please feel free to tag me directly for review in your pull request.
